test/index.ts

```diff
@@ -13,3 +18,40 @@ describe('Hello test', () => {
     expect(res.body.data.hello).to.be.eq('Hello, world!');
   });
 });
+
+describe('createUser test', () => {
+  it('createUser mutation', async () => {
```

Comment:
```suggestion
  it('should be possible to create user correctly', async () => {
```

cada it é um cenario de teste e com isso costumamos nomear de acordo com este cenario. 
Dito isso, seria interessante testar outros cenarios aqui, não? casos de erro tb (tentar inserir um usuario com email ja existente)

Reply:
Coloquei os casos de erro no PR "Feature/error handling #19"